### ISSUE TITLE
Case details | Add POA address

### ISSUE THREAD

**Issue by laurjpeterson** [department-of-veterans-affairs/caseflow#6812]
We want to show the Power of Attorney address in Case details to help BVA dispatchers (and likely other BVA staff) complete tasks that require mailing to an appellant's POA. 

## Mocks
- https://github.com/department-of-veterans-affairs/caseflow/issues/6812#issuecomment-419995260

## AC
- POA address from BGS exists in the POA section of Case details

**Comment by allyceh** [comment]
Adding what we've learned about POA to date. @laurjpeterson @amprokop , please confirm. cc: @sneha-pai @mkhandekar for design feedback. 

**What we know about POA:** 
- POA name and address for private attorneys: Lives in BGS
- POA name and address for VSOs: POA name lives in BGS, address pulled into the IDT and populated on decision letters lives in VACOLS
- POA info that lives in BGS must be edited in Share

Below are some mocks of how POA could look in Case Details. 

1) POA fields for private attorneys: 
<img width="820" alt="screen shot 2018-09-05 at 7 41 39 pm" src="https://user-images.githubusercontent.com/40216950/45126935-d76d7480-b143-11e8-8488-6d188539564d.png">

2) POA fields for VSOs: 
<img width="818" alt="screen shot 2018-09-05 at 7 41 53 pm" src="https://user-images.githubusercontent.com/40216950/45126953-f79d3380-b143-11e8-813f-c3a46456d673.png">

**Comment by amprokop** [comment]
What about cases with multiple POAs, or cases with more than one claimant with different POA? Are we going to wait on designing that? 

Eventually we'll have to transition to some model where POA is noted along with the person if there is more than one claimant— ok if we're saving that for later.

**Comment by laurjpeterson** [comment]
@amprokop - tracking showing multiple POA in this github issue. We have early sketches, but we did deprioritize incorporating the POA exceptions after we chatted with Shane about how Caseflow Intake might be the source of these POA exceptions, so we don't have data as we'd use it just yet - https://github.com/department-of-veterans-affairs/caseflow/issues/5589

**Comment by allyceh** [comment]
@mkhandekar I updated the verbiage to be more consistent and made it more concise (see below). 

<img width="408" alt="screen shot 2018-09-10 at 1 29 16 pm" src="https://user-images.githubusercontent.com/40216950/45313685-92f42700-b4fd-11e8-9826-83b0d2e52b02.png">
<img width="410" alt="screen shot 2018-09-10 at 1 29 06 pm" src="https://user-images.githubusercontent.com/40216950/45313688-938cbd80-b4fd-11e8-8d78-d669298a481e.png">

@laurjpeterson , if there is no additional feedback and given our October scope, these feel ready for development.

**Comment by laurjpeterson** [comment]
Looks good to me!

**Comment by mdbenjam** [comment]
Do we want to show the representative type? We currently do this (Attorney, VSO, etc.) or is that not needed.

**Comment by allyceh** [comment]
@mdbenjam I feel like that would be helpful. I am assuming we are able to clearly distinguish between private attorneys and VSOs. How about this, below: 

<img width="809" alt="screen shot 2018-09-20 at 11 36 52 am" src="https://user-images.githubusercontent.com/40216950/45829723-888a1800-bcc9-11e8-9813-06a6eafcd858.png">
<img width="807" alt="screen shot 2018-09-20 at 11 36 39 am" src="https://user-images.githubusercontent.com/40216950/45829726-8922ae80-bcc9-11e8-8ee0-717845548b6b.png">

I removed the word "representative". I don't think we are losing anything by not having this since we have the Power of Attorney label above.

**Comment by mdbenjam** [comment]
@allyceh  Cool, I think we should also distinguish between attorney and agent. So how about these options:

`Private Attorney`
`Private Agent`
`Veteran Service Organization` <- also we could split these into `Local/Regional Organization`, `State Organization`, and `National Organization` if that's useful.

**Comment by mdbenjam** [comment]
Actually it looks like we already have this mapping in the code:

    "POA Attorney" => "Attorney",
    "POA Agent" => "Agent",
    "POA Local/Regional Organization" => "Service Organization",
    "POA State Organization" => "Service Organization",
    "POA National Organization" => "Service Organization"

I'm going to keep using these on the right hand side. Omitting `Private` and `Veteran` for now for consistency. We can change this mapping if we want to make the change application wide.

**Comment by allyceh** [comment]
@mdbenjam For legacy and AMA appeals in Caseflow, is the POA data source BGS? Or, does the legacy POA come from VACOLS?

@laurjpeterson and I are looking to clarify.

**Comment by mdbenjam** [comment]
POA comes from BGS for AMA and VACOLS for legacy. At least for now. I'd like to eventually pull everything from BGS.

**Comment by laurjpeterson** [comment]
Ticket to add POA from BGS for legacy appeals - https://github.com/department-of-veterans-affairs/caseflow/issues/5601
